gulpfile.js

```diff
@@ -91,6 +91,7 @@ function buildExtensions(options) {
   buildExtension('amp-user-notification', '0.1', true, options);
   buildExtension('amp-vimeo', '0.1', false, options);
   buildExtension('amp-vine', '0.1', false, options);
+  buildExtension('amp-google-vrview-image', '0.1', false, options);
```

Comment:
So is the tag actually <amp-google-vrview-image>? Not to bikeshed more on the name, but seems really verbose :)